free-programming-interactive-tutorials-en.md

```diff
@@ -146,7 +146,7 @@
 ### NoSQL
 
 * [MongoDB Koans](https://github.com/chicagoruby/MongoDB_Koans)
-* [Try MongoDB](http://try.mongodb.org)
+* [Try MongoDB](https://university.mongodb.com/)
```

Comment:
Yep, I've removed this in https://github.com/davisonio/awesome-free-programming-books/commit/b61aef2ef82528338963ace92c0f53f00265d58c